cdap-api/src/main/java/co/cask/cdap/api/data/batch/Output.java

```diff
@@ -66,17 +67,16 @@ public Output alias(String alias) {
    *
    * @param datasetName the name of the output dataset
    */
-  public static Output ofDataset(String datasetName) {
+  public static DatasetOutput ofDataset(String datasetName) {
```

Comment:
I think it is. We do the same change for the `Input` class.

Al the usage of these `ofDataset` comes from the `addOutput()` method, which takes a `Output` as argument. Without this change, user will have to do an additional type conversion from `Output` -> `DatasetOutput` when specifying the namespace.